packages/crud/src/vaadin-crud.js

```diff
@@ -1232,14 +1232,18 @@ class Crud extends ControllerMixin(ElementMixin(ThemableMixin(PolymerElement)))
     this.__openEditor('edit', item);
   }
 
+  /** @private */
+  __ensureNotPrevented(event) {
+    const result = this.dispatchEvent(event);
+    return event.defaultPrevented === false || result;
```

Comment:
> false if event is cancelable, and at least one of the event handlers which received event called [Event.preventDefault()](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault). Otherwise true.
> https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent#return_value

Based on the above, I think the checks are pretty much equivalent, so it shouldn't be necessary to check for both.

Reply:
Done.